### PR TITLE
`FeatureFormView` - Address latest test failures

### PR DESCRIPTION
**Pull request by dfeinzimer**

### CODE REVIEW

File: Test Runner/UI Tests/FeatureFormViewTests.swift [hunk 1]
```diff
@@ -332,9 +332,15 @@ final class FeatureFormViewTests: XCTestCase {
...
-            Date.now.formatted()
+        // Verify that the date picker defaulted selection to the current date
+        // and time, accounting for the possibility that the minute has changed
+        // since the picker was opened.
+        XCTAssertTrue(
+            [
+                Date.now.formatted(),
+                Date.now.addingTimeInterval(-60).formatted()
+            ]
+                .contains(elementValue.label)
```
Comment (dfeinzimer) [inline Test Runner/UI Tests/FeatureFormViewTests.swift:343]: While this seems like an unlikely edge case, I've seen it happen a few times now, the latest being Sunday's daily run.

File: Test Runner/UI Tests/FeatureFormViewTests.swift [hunk 2]
```diff
@@ -1735,7 +1741,7 @@ final class FeatureFormViewTests: XCTestCase {
         let formTitle = app.staticTexts["Electric Distribution Device"]
         let lineEndCandidates = app.buttons.matching(identifier: "Line End")
         let lowVoltageSinglePhaseLineEnd = app.buttons["Low Voltage Single Phase Line End, Line End"]
-        let valueButton = app.buttons["Value"]
+        let valueButton = app.buttons["Value, Unknown"]
```
Comment (dfeinzimer) [inline Test Runner/UI Tests/FeatureFormViewTests.swift:1745]: As of #1405, the filter condition is correctly getting the first available option.

**Review by mhdostal — APPROVED** [review]

**Review by zkline101 — APPROVED** [review]
